.github/renovate.json5

```diff
@@ -0,0 +1,41 @@
+{
+  $schema: 'https://docs.renovatebot.com/renovate-schema.json',
+  extends: [
+    'config:recommended',
+  ],
+  ignoreUnstable: false,
+  packageRules: [
+    {
+      allowedVersions: '/^[0-9]+\\.[0-9]+(\\.[0-9]+)?$/',
+      matchPackageNames: [
+        '/.*/',
+      ],
+      automerge: true,
+      automergeStrategy: 'rebase',
+    },
+    {
+      matchFileNames: [
+        'gradle/stable.versions.toml',
+      ],
+      allowedVersions: '/^[0-9]+\\.[0-9]+(\\.[0-9]+)?$/',
+      automerge: true,
+      automergeStrategy: 'rebase',
+    },
+    {
+      matchFileNames: [
+        'gradle/alpha.versions.toml',
+      ],
+      allowedVersions: '/.*-alpha.*/',
+      automerge: true,
+      automergeStrategy: 'rebase',
+    },
+    {
+      matchFileNames: [
+        'gradle/beta.versions.toml',
+      ],
+      allowedVersions: '/.*-beta.*/',
+      automerge: true,
+      automergeStrategy: 'rebase',
+    },
```

Comment:
mind briefly explaining why separate updates?  is it because of google's release process or something else that i'm possibly unfamiliar with?

if it's due to the release process, then should we add one for rc releases as well?